redirect/routes.php

```diff
@@ -110,12 +110,20 @@ class Route {
         ],
         'tmmi'         => [
             'description' => 'Technik der Mensch inen Interaktion [Not Found]',
```

Comment:
Not found kann entfernt werden
```suggestion
            'description' => 'Technik der Menschinen Interaktion',
```